yarn-project/foundation/src/fields/fields.ts

```diff
@@ -412,7 +412,7 @@ export class Fq extends BaseField {
       return fromHexString(buf, Fq);
     }
 
-    throw new Error('Tried to create a Fq from an invalid string');
+    throw new Error(`Tried to create a Fq from an invalid string: ${buf}`);
```

Comment:
Improved the error message here as I needed it for debugging issue with slot value parsing.